tests/requirements_minimal.txt

```diff
@@ -6,4 +6,4 @@ boto3<1.11
 mock
 pytest-timeout==1.4.2
 argcomplete
-python-dateutil==1.5
+python-dateutil
```

Comment:
Due to the use of [tz.UTC](https://github.com/EnterpriseDB/barman/pull/475/files#diff-bf2bad4e300610a4cebe48674911e98fa7d7a819b133035fb14e8eff6130a0a3R29) there is a version dependency with python-dateutil and python 2. I'm not sure which version `tz.UTC` was added, but `python-dateutil==2.6.1` (the OS package version on Ubuntu 18.04) does not have it (whereas 2.8.1 does).

Reply:
Ok so it's 2.7.0 which introduced `tz.UTC`: https://dateutil.readthedocs.io/en/stable/tz.html#dateutil.tz.dateutil.tz.UTC

I think this can be resolved by using `tz.tzutc()` in place of `tz.UTC`.

Reply:
indeed, tz.UTC exist since 2.7.0.  I actually encounter some issues with barman testing. python-dateutil is installed with apt-get wich provides 2.6.1.  trying pip install  `python-dateutil>=2.7.0` does not change anything whereas  `python-dateutil==2.7.0` does the trick but I find it a bit limiting.
Not sure how to solve this properly yet.

I will have a look at tz.tzutc()

Reply:
So tz.tzutc() seems to work fine. I will use it.